Fix Arch Migrator issues from PR #12

georgemccabe opened the pull request:
Support for python 3.14 was inadvertently removed when adding support for arch

**NOTE: until the issue is resolved with the migrators, rerendering with conda-smithy will remove support for python 3.14**

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

georgemccabe committed:
resolves issues from conda-forge/nceplibs-bufr-feedstock#12 where support for python 3.14 was inadvertently removed when adding support for arch -- NOTE: until the issue is resolved with the migrators, rerendering with conda-smithy will remove support for python 3.14

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

georgemccabe commented:
Note: I did not add python 3.14 for the new `linux_aarch64` and `linux_ppc64le` build, because they weren't auto generated (obviously) and I wasn't sure if they supported 3.14 yet or not.

georgemccabe committed:
merged main and resolve conflicts

georgemccabe committed:
MNT: Re-rendered with conda-smithy 3.55.1 and conda-forge-pinning 2026.03.05.07.27.18

Other tools:
- conda-build 26.1.0
- rattler-build 0.46.0
- rattler-build-conda-compat 1.4.5

georgemccabe committed:
Merge remote-tracking branch 'origin/main' into fix_arch_migrator

georgemccabe committed:
increment build number

Commit:
MNT: Re-rendered with conda-smithy 3.55.1 and conda-forge-pinning 2026.03.06.10.44.2
